homeassistant/config_entries.py

```diff
@@ -347,6 +348,15 @@ def async_entries(self, domain=None):
         await entry.async_setup(
             self.hass, component=getattr(self.hass.components, component))
 
+    async def async_forward_entry_unload(self, entry, component):
```

Comment:
The opposite of setup is teardown. The opposite of unload is load. Maybe we should pick one of those pairs for names?

Reply:
You sir, make an excellent point.

Reply:
`async_tear_down_entry` just sounds so aggressive 🤔

Reply:
(I want to stick with `setup` because we now have `setup` and `setup_entry` and that shows they are related)

Reply:
Removing sounds like we're deleting the entry instead of just undoing the setup of  it.

Reply:
`revoke` ?